### ISSUE TITLE
Abnormal body image

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11654]
# Abnormal body image

## Proposed definition

A cognitive distortion characterized by a persistently distorted perception or evaluation of one's own body shape, size, or appearance.

## Parent term

- `HP:5200242 Cognitive distortion`

## Synonyms

- Body image disturbance
- Distorted body image

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

---

**Comment by pnrobinson** [comment]
@mellybelly  @evanc1648
@bcoleman96 

Evan, often this term is used in the context of eating disorders. We can craft a definition as follows

**def**: A distortion in the perception, evaluation, or experience of one's own body, resulting in a body image that is inaccurate, distorted, or excessively influenced by body shape or weight.
**Comment**: Body image is the intuitive portrayal a person holds of their body, regardless of their appearance. Body image is, thus, a complex concept including self-observations, cognitions, emotions, memories, fantasies as well as behaviors related to an individual’s body, both conscious and unconscious. Body image is how an individual considers or imagines their body and how it appears to others. Gender and social standards might influence the ideal physique. A person has a poor body image when they are preoccupied with the fictitious flaws of their physical appearance.
PMID: [38102717](https://pubmed.ncbi.nlm.nih.gov/38102717/)

We want to have term definitions that apply across the entire disease/HPO spectrum. I think the key part of the definition is "fictitous flaws" rather than "concerns about body appearance" (which would be a typical response to some physical features of some genetic disorders).  So while I think we should add these terms, you may want to consider if the term defined as above applies to FA?

I would add synonyms:
-Body image disturbance
-Distorted body image
-Disturbed body image

**Comment by evanc1648** [comment]
Thank you, @pnrobinson! It all looks great. I agree on adding the term.

On your FA question--I traced our source finding ("body image concerns") to the Clinical Care Guidelines ([Chapter 7](https://fanconi.org/clinical-care-guidelines/gynecologic-care-for-female-patients-with-fanconi-anemia/)), from this passage:

> "…women with chronic medical conditions may be at increased risk of depression, body image concerns, or social isolation, which also can impact sexual function."

Therefore, "body image concerns" here is a generic psychosocial risk of chronic illness, listed alongside depression and social isolation. It's framed for women with chronic medical conditions generally (not FA-specific). This matches your point: this is concern about real circumstances, not fictitious-flaws.

Let's add the term, but we'd not annotate it to FA, since our source doesn't describe the phenotype. Thanks for catching it.

**Comment by pnrobinson** [comment]
@evanc1648 sounds good!
I am adding the term
Adding new term: 
- ID: HP:6001533
- label: Abnormal body image
- parents: [Cognitive distortion (HP:5200242)](https://hpo.jax.org/app/browse/term/HP:5200242)
- pmid: PMID:38102717
- def: A distortion in the perception, evaluation, or experience of one's own body, resulting in a body image that is inaccurate, distorted, or excessively influenced by body shape or weight.
- comment: Body image is the intuitive portrayal a person holds of their body, regardless of their appearance. Body image is, thus, a complex concept including self-observations, cognitions, emotions, memories, fantasies as well as behaviors related to an individual's body, both conscious and unconscious. Body image is how an individual considers or imagines their body and how it appears to others. Gender and social standards might influence the ideal physique. A person has a poor body image when they are preoccupied with the fictitious flaws of their physical appearance.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11654
- synonyms: Distorted body image(NONE)|Disturbed body image(NONE)|Body image disturbance(NONE)
